src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 900;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
```

Comment:
You have the same names of two different fonts, please give them different names and use depending on the condition of your task
Also check your code through tests, you have 5 failed tests. They must be corrected